de_web_plugin.cpp

```diff
@@ -5059,7 +5059,8 @@ void DeRestPluginPrivate::addSensorNode(const deCONZ::Node *node, const deCONZ::
                     {
                         fpCarbonMonoxideSensor.inClusters.push_back(IAS_ZONE_CLUSTER_ID);
                     }
-                    else if (!node->nodeDescriptor().isNull() && node->nodeDescriptor().manufacturerCode() == VENDOR_NONE)
+                    else if ((!node->nodeDescriptor().isNull() && node->nodeDescriptor().manufacturerCode() == VENDOR_NONE) ||
+                               node->nodeDescriptor().manufacturerCode() == VENDOR_EMBER)
```

Comment:
I wonder if the nodeDescriptor checks in the `else if` should be simplified to :

```cpp
else if (isTuyaManufacturerName(manufacturer))
{
}
```

Does this makes sense?

Reply:
Yes for the moment all device are from tuya manufacture, so no problem, have modified.
In my mind was faster to compare 2 manufactures number, than use regex.

Reply:
Thanks, faster yes but even on the RPi I expect the code to run in less than 1ms :)
It's also only run once during setup.